README.md

```diff
@@ -111,6 +111,20 @@ as well as its correspondent options.
 python cli.py gen_swagger_json -f ~/Downloads/swagger.json
 ```
 
+## Run as docker container
+
+1. Build image
+```bash
+docker build -t time_tracker_api:local .
+```
+
+2. Run app
+```bash
+docker run -p 5000:5000 time_tracker_api:local
+```
+
+3. Visit `0.0.0.0:5000`
```

Comment:
Actually he needs to visit localhost or 127.0.0.1. The IP `0.0.0.0` makes sense inside docker but outside you will get this error
![image](https://user-images.githubusercontent.com/6514740/77125311-70589d80-6a13-11ea-8e10-86a154820939.png)